### PR TITLE
Image server

### PR DESCRIPTION
**Pull request by goastler**

### CODE REVIEW

**Comment by goastler** [comment]
fixes prosopo/captcha-private#609

File: packages/file-server/env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+REMOTES="http://localhost:3001" # list of remote servers to proxy requests to (can be JSON array)
```
Comment (forgetso) [inline packages/file-server/env.example:1]: I hi

File: packages/file-server/env.example [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+PORT=3000 # port to run the server on
```
Comment (forgetso) [inline packages/file-server/env.example]: I think the default port should be in line with the rest of our ports. We've used `9228` to `9231` so this one should perhaps be `9232`

**Review by forgetso — APPROVED** [review]

**Review by forgetso — CHANGES_REQUESTED** [review]
Looks good. Just one very small env file query.

File: packages/file-server/env.example (hunk 2 above)
Comment (goastler) [reply packages/file-server/env.example]: good shout, I agree :)

**Review by forgetso — APPROVED** [review]

**Pull request merged**